README.md

```diff
@@ -15,9 +15,9 @@ Topics for discussion are suggested by [the gap-analysis template](https://www.w
 ---
 
 ## Help wanted!
-**We're looking for information about this writing system. Follow the link for specific questions.**
+**We're looking for information about this writing system. [Follow the link](https://github.com/w3c/clreq/issues?q=is:issue+is:open+label:%E5%95%8F%E9%A1%8C/question) for specific questions.**
```

Comment:
minor nit: "Follow this link" may be better.

Reply:
Thanks! Updated.